jdbc/pom.xml

```diff
@@ -0,0 +1,81 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring-data-jdbc-examples</artifactId>
+    <packaging>pom</packaging>
+
+    <parent>
+        <groupId>org.springframework.data.examples</groupId>
+        <artifactId>spring-data-examples</artifactId>
+        <version>2.0.0.BUILD-SNAPSHOT</version>
+    </parent>
+
+    <name>Spring Data JDBC - Examples</name>
+    <description>Sample projects for Spring Data JDBC</description>
+    <url>http://projects.spring.io/spring-data-jdbc</url>
+    <inceptionYear>2017</inceptionYear>
+
+    <modules>
+        <module>basics</module>
+    </modules>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.springframework.data</groupId>
+            <artifactId>spring-data-jdbc</artifactId>
+            <version>1.0.0.BUILD-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.data</groupId>
+            <artifactId>spring-data-commons</artifactId>
+            <version>2.1.0.BUILD-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
```

Comment:
I think that's been pulled in via the parent already.

Reply:
No, HSQLDB does not get pulled in through the parent.